wallet/rpc/rpc.proto

```diff
@@ -40,6 +32,5 @@ message SendFilResponse {
 service RPCService {
     rpc NewAddress(NewAddressRequest) returns (NewAddressResponse) {}
     rpc List(ListRequest) returns (ListResponse) {}
-    rpc Balance(BalanceRequest) returns (BalanceResponse) {}
```

Comment:
Moved this into `powergate.proto` just to see how it feels. Feels good! You'll see this change reflected in the Go client code.